syntax = "proto3";

package text_update;

service TextUpdateService {
  rpc StreamTextUpdates (TextUpdateRequest) returns (stream TextUpdateResponse) {}
}

message TextUpdateRequest {
  uint32 client_id = 1;
}

message TextUpdateResponse {
  string updated_text = 1;
  uint32 updated_number = 2;
}
